lib/forest_liana/schema_file_updater.rb

```diff
@@ -96,6 +97,13 @@ def initialize filename, collections, meta
         end
 
         collection['actions'] = collection['actions'].map do |action|
+          begin
+            SmartActionFieldValidator.validate_smart_action_fields(action['fields'], action['name'], action['hooks']['change'])
```

Comment:
I think the validation of the hook definition is missing here

Reply:
Interesting one. Due to the way you declare the smart action hook I think.
I tend to use `String` instead of Symbol to match the `forest-express-sequelize` contribution (And the existing code in this file), but I wonder what's the best practice here 🤔